### ISSUE TITLE
The training loss curve during the fine-tuning stage

### ISSUE THREAD

**Issue by CnFaker** [JiuhaiChen/BLIP3o#82]
Dear author, thank you for your interesting work. Could you share approximately what the loss was during the fine-tuning stage? I’d like to know whether my training is converging normally.  I noticed that the wandb link you shared in another issue is no longer accessible. Thank you a lot！

**Comment by JiuhaiChen** [comment]
Hi, we have the training curve for BLIP3o-NEXT pretraining: https://api.wandb.ai/links/jchen169/kb1n4no8, 
you need curve for BLIO3o?

**Comment by CnFaker** [comment]
Thank you for sharing blip3o-next. It would be even better if there were blip3o siglip sana.



发自我的iPhone


------------------ Original ------------------
From: JiuhaiChen ***@***.***&gt;
Date: Thu,Sep 18,2025 3:35 AM
To: JiuhaiChen/BLIP3o ***@***.***&gt;
Cc: Big Fly ***@***.***&gt;, Author ***@***.***&gt;
Subject: Re: [JiuhaiChen/BLIP3o] The training loss curve during thefine-tuning stage (Issue #82)



JiuhaiChen left a comment (JiuhaiChen/BLIP3o#82)
 
Hi, we have the training curve for BLIP3o-NEXT pretraining: https://api.wandb.ai/links/jchen169/kb1n4no8,
 you need curve for BLIO3o?

**Comment by orrzohar** [comment]
Hi @JiuhaiChen !

Can you share Blip3o training curves too?

**Comment by JiuhaiChen** [comment]
Sure, we have the issue for training curve. 
https://github.com/JiuhaiChen/BLIP3o/issues/16
